src/app/auth/resetPassword/styles.tsx

```diff
@@ -0,0 +1,29 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    paddingVertical: 64,
+    paddingLeft: 44,
+    paddingRight: 44,
+    flex: 1,
+    justifyContent: 'space-between',
+  },
+
+  newPassword: {
+    paddingBottom: 8,
```

Comment:
You should add a padding top here of 8px. According to the designs, there should be 24px between the header and the new password textbox, and since the textbox component has 16px above it already we need to add 8px more.